src/scales/scale.time.js

```diff
@@ -146,16 +146,26 @@ module.exports = function(Chart) {
 			minUnit: 'millisecond',
 
 			// defaults to unit's corresponding unitFormat below or override using pattern string from http://momentjs.com/docs/#/displaying/format/
+			// Use more detailed date strings on major ticks and summary info on minor ticks
 			displayFormats: {
-				millisecond: 'h:mm:ss.SSS a', // 11:20:01.123 AM,
-				second: 'h:mm:ss a', // 11:20:01 AM
-				minute: 'h:mm a', // 11:20 AM
+				millisecond: '.SSS', // .123,
```

Comment:
I don't recall we changed them in 2.6.0: it a breaking change since it will impact every projects that didn't explicitly override **all** `displayFormats` values, which I guess is many projects.